OpenRA.Mods.Common/Traits/BotModules/SquadManagerBotModule.cs

```diff
@@ -119,7 +123,16 @@ public SquadManagerBotModule(Actor self, SquadManagerBotModuleInfo info)
 			unitCannotBeOrdered = a => a == null || a.Owner != Player || a.IsDead || !a.IsInWorld;
 		}
 
-		public bool IsEnemyUnit(Actor a)
+		// Use for proactive targeting.
+		public bool IsPreferredEnemyUnit(Actor a)
+		{
+			if (a == null || a.IsDead || Player.Stances[a.Owner] != Stance.Enemy || a.Info.HasTraitInfo<HuskInfo>())
+				return false;
```

Comment:
Minor style nit: I know we still have some places like that left in the code, but generally we want an empty line after something like an if-statement if it's followed by something else.

Reply:
I added the missing empty line.